### PR TITLE
Post: collaborative clusters

### PR DESCRIPTION
**Pull request by hsanjuan**

### CODE REVIEW

File: content/post/076-collaborative-clusters.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Instructions on how to setup and join these and, in the future, other
+collaborative clusters can be found at
+[https://collab.ipfscluster.io](https://collab.ipfscluster.io). You can stop
+and re-start replication whenever you want. Also, here's a quick video to show
+how easy it is:
+
+<script id="asciicast-yV2Bk4nlrPAQ6MQ4w6z3ea0uZ" src="https://asciinema.org/a/yV2Bk4nlrPAQ6MQ4w6z3ea0uZ.js" async></script>
+
+We hope that collaborative clusters will allow the community to participate in
+distributiong and seeding of data they care about, enabling easy "mirroring"
```
Comment (momack2) [inline content/post/076-collaborative-clusters.md]:
```suggestion
the distribution and seeding of data they care about, enabling easy "mirroring"
```

**Comment by hsanjuan** [comment]
Are we ready with this?

File: content/post/076-collaborative-clusters.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+We are very excited to announce the first set of public *collaborative
+clusters* and the first iteration of the
+[collaborative clusters website](https://collab.ipfscluster.io).
+
+*Collaborative clusters* are an easy way to join and help improving distribution
+and data availability of specific datasets in the IPFS Network.
+
+Using IPFS Cluster's
```
Comment (momack2) [inline content/post/076-collaborative-clusters.md]: I feel like this is missing some sort of visual or way of communicating the value of a collaborative community cluster. Maybe not blocking here - but try opening a design shop request to @akrych or @ericronne to get their help landing this? Could be a nice visual to also include in the website going forward!

Comment (hsanjuan) [reply content/post/076-collaborative-clusters.md]: Agreed, I will draft something

**Pull request merged**